cdap-common/src/main/java/io/cdap/cdap/common/http/AuthenticationChannelHandler.java

```diff
@@ -123,4 +122,16 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
     HttpUtil.setKeepAlive(response, false);
     ctx.channel().writeAndFlush(response).addListener(ChannelFutureListener.CLOSE);
   }
+
+  private void resetCurrentUserInfo() {
+    if (internalAuthEnabled) {
+      currentUserID = EMPTY_USER_ID;
+      currentUserCredential = new Credential(EMPTY_USER_CREDENTIAL, Credential.CredentialType.INTERNAL);
```

Comment:
Seems like the `Credential` object can be a constant too?

Reply:
Fixed.